data/human/human missions.txt

```diff
@@ -2514,6 +2514,139 @@ event "terraforming follow-up"
+			`	"I just need to tend to a few things here, in preparation for my absence, and I'll be at your ship in a few hours," she says.`
+			`	Once she arrives, you show her to her bunk and she settles in for the journey.`
+				accept
+	on complete
+		conversation
+			`As you approach <planet>, Amy is astounded by the magnitude of volcanic ash in the air, far exceeding what was released by the volcano you triggered on Tundra. "This place makes Tundra seem temperate," she remarks after observing the glaciers that occupy most of the planet's polar regions.`
+			`	Her fascination with the state of this world is temporarily forgotten when she exits your ship and sees the Wanderers for the first time. "I don't know what I expected them to be like, but it wasn't birds," she mumbles, staring at them from head to toe. However, she quickly returns to the reason for coming here. "Do you know how we can find someone who can answer my questions?"`
+			`	Approaching one of the nearby Wanderers, you ask if they could direct you to Tema'a Iriket, the terraforming director. While they don't appear to understand the Hai language from your translator, they seem to recognize the name and gesture for you to wait in the spaceport, before flying off into the distance, presumably to fetch Iriket for you.`
+
+
+
+mission "Terraforming 14"
+	name "Field Trip"
+	description "Take Tema'a Iriket and Amy to <destination> to show her a world where the Wanderers have finished their work."
+	source "Desi Seledrak"
+	destination "Var' Kayi"
+	passengers 2
+	blocked "You need <capacity> in order to take on this mission."
+	to offer
+		has "Terraforming 13: done"
+	on offer
+		conversation
+			`It is not long before Iriket arrives. You introduce Amy to her, and explain why you have brought her here. "I would be [glad, honored] to answer any questions about our work," she agrees.`
+			`	What follows is an extensive discussion about how the Wanderers go about planning the terraforming process for a world. Topics range from the earliest considerations as to what the goal for each planet should be to the role of their artificial minds in helping to monitor and guide the process in the long term. Your translator has some difficulty with the more technical aspects of the discussion, which leads Iriket to rephrase some things many times. Throughout it all, Amy is taking copious notes. Iriket also explains some of what they have been doing to improve the climate on this world.`
+			`	"The gases [released, produced] by the factories will help the atmosphere retain more heat, which will in turn begin to melt the glaciers and return water to the [ecosystem, environment]. Over time, this will [mix, interact] with the abundant ash, helping to [restore, rejuvenate] the soils all over this world. In time, it will become [fruitful, bountiful] again."`
```

Comment:
```suggestion
			`	"The gasses [released, produced] by the factories will help the atmosphere retain more heat, which will in turn begin to melt the glaciers and return water to the [ecosystem, environment]. Over time, this will [mix, interact] with the abundant ash, helping to [restore, rejuvenate] the soils all over this world. In time, it will become [fruitful, bountiful] again."`
```
American spelling

Reply:
Source? It seems like "gasses" is usually used for the verb form, and the plural noun is usually "gases". I can't find evidence that the American standard for the noun form is "gasses".

Reply:
Further investigation: ~~why did I take my source from only two websites~~ this can be ignored.